app/uk.gov.hmrc.helptosavestridefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -150,6 +156,26 @@ class HelpToSaveConnectorImpl @Inject() (http:                              WSHt
           }
       })
 
+  override def createAccount(nSIUserInfo: NSIUserInfo)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[CreateAccountResult] = {
+
+    EitherT[Future, String, CreateAccountResult](http.post(createAccountUrl, nSIUserInfo).map[Either[String, CreateAccountResult]] { response ⇒
+      response.status match {
+        case Status.CREATED ⇒
+          logger.info(s"createAccount returned 201 (Created) with nino: ${nSIUserInfo.nino}")
```

Comment:
I have changed the logger lines to be like (s"blah blah", nino) so that they all use the NINOLogMessageTransformer and there isnt currently a def debug in Logging